pkg/build/builder/docker_test.go

```diff
@@ -117,11 +117,11 @@ func cmpASTs(original *parser.Node, edited *parser.Node) int {
 		index++
 	}
 
-	originalChildren := make([]*parser.Node, 0)
+	var originalChildren []*parser.Node
```

Comment:
I didn't have an eye on optimizing it... just golint. If you take a broader look, this code shouldn't even be there. I think we can nuke both `originalChildren` and `editedChildren` altogether.

But leave this aside for a while, let's look back at it after I refactor the old code to use the new functions.